magento/securify
# Recaptcha initialization

### Description (*)

This PR accumulates commits from https://github.com/magento/security-package/pull/183,  https://github.com/magento/security-package/pull/176 and https://github.com/magento/security-package/pull/173

### Fixed Issues (if relevant)
https://github.com/magento/security-package/issues/21

### Manual testing scenarios (*)

1. See https://github.com/magento/security-package/pull/183
2. See https://github.com/magento/security-package/pull/176
3. See https://github.com/magento/security-package/pull/173

### Questions or comments


### Contribution checklist (*)
